### PR TITLE
Respondkey Changing in UI

### PR DESCRIPTION
**Pull request by LeSiiN**
[WIP] Respondkey Changing in UI
https://www.youtube.com/watch?v=3BpBt3JXUiw

### CODE REVIEW

File: ui/src/components/Main.svelte [hunk 1]
```diff
@@ -32,6 +33,12 @@
     });
   });
 
+  window.addEventListener('message', (event) => {
```
Comment (OK1ez) [inline ui/src/components/Main.svelte]: Use this insted https://github.com/Project-Sloth/ps-dispatch/blob/3988e28ec86df76b9ff51e465a7c1a2e9145e52b/ui/src/providers/AlwaysListener.svelte#L48

File: ui/src/components/Main.svelte [hunk 2]
```diff
@@ -86,7 +93,7 @@
           <div class="w-[30%] flex items-end justify-center mb-[1vh]">
             {#if index === 0}
               <p class="px-[1.5vh] py-[0.4vh] rounded-full bg-primary text-[1.3vh]">
-                [E] Respond
+                [{RespondKeybind}] Respond
```
Comment (OK1ez) [inline ui/src/components/Main.svelte]: In store.ts make a new store export const RESPOND_KEYBIND = writable<string>("");

Insted of [{RespondKeybind}] it should be [{$RESPOND_KEYBIND }] , note: you have to import the RESPOND_KEYBIND to the svelte file

File: ui/src/components/Main.svelte (hunk 1 above)
Comment (OK1ez) [reply ui/src/components/Main.svelte]: The reciveNUI should look like this, remember to import the new RESPOND_KEYBIND store

	ReceiveNUI('setupUI', (data: any) => {
		PLAYER.set(data.player)
		Locale.set(data.locales)
	        RESPOND_KEYBIND .set(data.locales)
	});

Comment (LeSiiN) [reply ui/src/components/Main.svelte]: done

File: ui/src/components/Main.svelte (hunk 2 above)
Comment (LeSiiN) [reply ui/src/components/Main.svelte]: done

**Comment by 5tratz** [comment]
Tested and working fine for me :)

**Comment by MonkeyWhisper** [comment]
Tested and approved

**Pull request merged**